### PR TITLE
AuditManager

### PR DESCRIPTION
**Pull request by BioEnable-bit**
1. AuditManagerSerivceImpl is created 
2. AuditManagerService is created 
3. AuditDao is created

### CODE REVIEW

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/AuditManagerSerivceImpl.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
+package io.mosip.registration.clientmanager.spi.audit;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/AuditManagerSerivceImpl.java:1]: Impl classes should be added directly under package: io.mosip.registration.clientmanager.service

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/builder/AuditRequestBuilder.java [hunk 2]
```diff
@@ -0,0 +1,233 @@
+package io.mosip.registration.clientmanager.spi.audit.builder;
+
+import java.time.LocalDateTime;
+
+import org.springframework.stereotype.Service;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/builder/AuditRequestBuilder.java:5]: invalid imports must be removed

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/request/AuditRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
+package io.mosip.registration.clientmanager.spi.audit.request;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/request/AuditRequestDto.java:1]: dtos must be added under package : io.mosip.registration.clientmanager.dto

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/AuthTokenUtilService.java [hunk 4]
```diff
@@ -0,0 +1,412 @@
+//package io.mosip.registration.clientmanager.util.restclient;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/AuthTokenUtilService.java:1]: this file can be deleted

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/RequestHTTPDTO.java [hunk 5]
```diff
@@ -0,0 +1,164 @@
+//package io.mosip.registration.clientmanager.util.restclient;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/RequestHTTPDTO.java:1]: this file can be deleted

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/RestClientUtil.java [hunk 6]
```diff
@@ -0,0 +1,166 @@
+//`package io.mosip.registration.clientmanager.util.restclient;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/RestClientUtil.java:1]: this file can be deleted

File: client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/ServiceDelegateUtil.java [hunk 7]
```diff
@@ -0,0 +1,418 @@
+//package io.mosip.registration.clientmanager.util.restclient;
```
Comment (anshulv1401) [inline client/clientmanager/src/main/java/io/mosip/registration/clientmanager/util/restclient/ServiceDelegateUtil.java:1]: this file can be deleted

**Review by anshulv1401 — CHANGES_REQUESTED** [review]
Please check the comments for the changes required

**Review by BioEnable-bit — COMMENTED** [review]
@anshulv1401 Given changes are done. Please check at your side.

**Comment by ase-101** [comment]
Closing as the PR is outdated.

**Pull request closed without merging**